views/file_list.erb

```diff
@@ -37,9 +39,15 @@
             <%= link_to_source_file(source_file) %>
           </td>
           <td class="<%= coverage_css_class(source_file.covered_percent) %> strong"><%= source_file.covered_percent.round(2).to_s %> %</td>
+          <!-- <td class="<%= "#{coverage_css_class(source_file.covered_percent)}_runtime" %> strong"><%= source_file.covered_percent.round(2).to_s %> %</td> -->
           <td><%= source_file.lines.count %></td>
           <td><%= source_file.covered_lines.count + source_file.missed_lines.count %></td>
           <td><%= source_file.covered_lines.count %></td>
+          <td>
+            <%= result.from_type(Coverband::RUNTIME_TYPE) do
```

Comment:
Probably an opportunity to introduce a method into `CoverbandResults` to do this?
```erb
<%= result.covered_lines(source_file.filename, type: Coverband::RUNTIME_TYPE) %>
```

Although it would feel more consistent to just have that on source_file.

```erb
<%= source_file.covered_lines(type: Coverband::RUNTIME_TYPE).count %>
```

Reply:
now like this `<%= result.file_with_type(source_file, Coverband::RUNTIME_TYPE).try(:covered_lines).try(:count) || 0 %>` which is better than it was, I don't love needing to use `try` as not all files or lines will exist in runtime coverage.